sycl/test-e2e/CMakeLists.txt

```diff
@@ -58,6 +64,7 @@ if(SYCL_TEST_E2E_TARGETS)
     add_custom_target(${TARGET}
       COMMAND ${Python3_EXECUTABLE} ${LLVM_LIT} ${SYCL_E2E_TESTS_LIT_FLAGS} --param sycl_be=${TARGET_BE} --param target_devices=${TARGET_DEVICES} .
       COMMENT "Running the SYCL tests for ${TARGET} backend"
+      DEPENDS ${SYCL_E2E_TEST_DEPS}
```

Comment:
I'm not sure how CMake reacts to an "uninitialized variable", let's see what CI will tell us.

Reply:
According [to the documentation](https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#variable-references):

> A variable reference has the form ${<variable>} and is evaluated inside a [Quoted Argument](https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#quoted-argument) or an [Unquoted Argument](https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#unquoted-argument). A variable reference is replaced by the value of the specified variable or cache entry, or if neither is set, by the empty string.

Reply:
Thanks! I also see some testing has successfully finished, so the patch looks good.

Thanks for taking care of this!

PS, there might be one other issue with this non-standalone configuration where we inherit the compilation flags from the project, but I didn't have cycles to look into that, so JFYI.